void tomo::IoLoop::run_loop<false, true, false, true, false>()
--- PRE

+++ POST

@@ -532,7 +532,7 @@

 0a1b [ 5] jmp <+0x4b0>
 0a20 [ 5] mov $0x1,%esi
 0a25 [ 3] mov %rbx,%rdi
-0a28 [ 5] call <tomo::IoLoop::arm_accept(tomo::UrKind)>
+0a28 [ 5] call <tomo::IoLoop::fliplisten_retry_accept(tomo::UrKind)>
 0a2d [ 5] jmp <+0x2d2>
 0a32 [11] data16 cs nopw 0x0(%rax,%rax,1)
 0a3d [ 3] nopl (%rax)
